### PR TITLE
Update Weights for Claims Pallet

### PR DESCRIPTION
**Pull request by shawntabrizi**
This PR updates the weights of the claims pallet to use the benchmarking data collected for it.

### CODE REVIEW

File: runtime/common/src/claims.rs [hunk 1]
```diff
@@ -292,6 +302,10 @@ impl<T: Trait> sp_runtime::traits::ValidateUnsigned for Module<T> {
 		const PRIORITY: u64 = 100;
 
 		match call {
+			// <weight>
+			// Base Weight: 370 µs
+			// DB Weight: 1 Read (Claims)
+			// </weight>
```
Comment (thiolliere) [inline runtime/common/src/claims.rs:308]: So validate_unsigned time computation is not part of claim weight because it less than ExtrinsicBaseWeight ?

Comment (shawntabrizi) [reply runtime/common/src/claims.rs:308]: Yes, and we noted that in our Riot chat. I think this is supposed to be "good enough for now".

I think though, to be safe, I should include this weight into the main extrinsic until we polish all the details.

Comment (thiolliere) [reply runtime/common/src/claims.rs:308]: yes, IMHO currently ExtrinsicBaseWeight to be 10µs so I would say better to put into the extrinsic as well. In the future maybe we should have SignedExtrinsicBaseWeight UnsignedExtrinsicBaseWeight.

(Also the benchmark could include both so read/writes are not duplicated (but here its fine))

Comment (shawntabrizi) [reply runtime/common/src/claims.rs:308]: > In the future maybe we should have SignedExtrinsicBaseWeight UnsignedExtrinsicBaseWeight.

yes, I think that will be the case

**Pull request merged**